client/src/components/Common/DetailsLabel/constants.js

```diff
@@ -0,0 +1,55 @@
+export const TYPES = {
+  expences: 'EXPENCES',
+  duration: 'DURATION',
+  rating: 'RATING',
+};
+
+const colors = {
+  very_good: '#00B300',
+  good: '#7FBB17',
+  ok: '#FEC22D',
+  bad: '#F47048',
+  very_bad: '#E91E63',
+};
+
+const limitsExpences = [100, 500, 1000, 5000];
+const limitsDuration = [2, 7, 14, 31];
+const limitsRating = [1, 2, 3, 4];
+
+const getColorsInversed = (value, limits) => {
```

Comment:
It is not constants :)
Move these functions to helpers.

Reply:
Well, it is true... But as I would like to keep all this logic in the same file, I am going to rename all the file to "constants.js" ---> "utils.js". 

What do you think?

Reply:
Yes, good idea! I think, your new solution is more appropriate.